src/path/Path.js

```diff
@@ -2123,6 +2123,42 @@ var Path = PathItem.extend(/** @lends Path# */{
      * the beginning of the path and {@link Path#length} at the end
      * @return {Number} the normal vector at the given offset
      */
+
+    /**
+     * Calculates path-time parameters where the path is tangent to given vector.
```

Comment:
path-time is a misleading concept that doesn't exist elsewhere in the library. Note that curve-time isn't linear, a curve at t = 0.5 is most of the time no the middle of the curve (at half of its length), because there is tension in curves. path-time here is an emulation where length = 1, 0 = 0, and 0.5 = length / 2.  I think it's better to just speak of lengths instead:

`path.getLengthsWithTangent()`

Reply:
Ok, I understand what you say, so I guess it should return a list of lengths instead of a list of "times" between 0 and 1, right ?
And maybe we could speak of offsets instead of lengths as in other path methods ex : [path.getOffsetOf()](http://paperjs.org/reference/path/#getoffsetof-point) ?
`path.getOffsetsWithTangent()`

Reply:
Yes you're right! That's the right name for it!